tests/text/test_wil.py

```diff
@@ -0,0 +1,70 @@
+from typing import List, Union
+
+import pytest
+from jiwer import wil
+
+from tests.text.helpers import INPUT_ORDER, TextTester
+from torchmetrics.functional.text.wil import word_information_lost
+from torchmetrics.text.wil import WordInfoLost
+from torchmetrics.utilities.imports import _JIWER_AVAILABLE
+
+BATCHES_1 = {"preds": [["hello world"], ["what a day"]], "targets": [["hello world"], ["what a wonderful day"]]}
+
+BATCHES_2 = {
+    "preds": [
+        ["i like python", "what you mean or swallow"],
+        ["hello duck", "i like python"],
+    ],
+    "targets": [
+        ["i like monthy python", "what do you mean, african or european swallow"],
+        ["hello world", "i like monthy python"],
+    ],
+}
+
+
+def _compute_wil_metric_jiwer(prediction: Union[str, List[str]], reference: Union[str, List[str]]):
+    return wil(reference, prediction)
+
+
+@pytest.mark.skipif(not _JIWER_AVAILABLE, reason="test requires jiwer")
+@pytest.mark.parametrize(
+    ["preds", "targets"],
+    [
+        pytest.param(BATCHES_1["preds"], BATCHES_1["targets"]),
+        pytest.param(BATCHES_2["preds"], BATCHES_2["targets"]),
```

Comment:
```suggestion
        (BATCHES_1["preds"], BATCHES_1["targets"]),
        (BATCHES_2["preds"], BATCHES_2["targets"])
```
just tuple is fine if you don't use pytest argument